exist-parent/pom.xml

```diff
@@ -1228,13 +1239,24 @@
                 <enabled>true</enabled>
             </snapshots>
         </pluginRepository>
+        <pluginRepository>
+            <id>github</id>
+            <name>GitHub Registry for Snapshots</name>
+            <url>https://maven.pkg.github.com/eXist-db/exist</url>
+            <releases>
+                <enabled>false</enabled>
+            </releases>
+            <snapshots>
+                <enabled>true</enabled>
+            </snapshots>
+        </pluginRepository>
     </pluginRepositories>
 
     <distributionManagement>
         <snapshotRepository>
-            <id>exist-db-snapshots</id>
-            <name>Evolved Binary - eXist-db Snapshots</name>
-            <url>https://repo.evolvedbinary.com/repository/exist-db-snapshots/</url>
+            <id>github</id>
```

Comment:
Please also restore the old entry for `Evolved Binary - eXist-db Snapshots` as this is needed for the nightly builds and releases.

Reply:
@adamretter see my earlier comment. There can only be one `snapshotRepository`

Reply:
@duncdrum So how can we fix this then?